installer/steps/codex_files.py

```
@@ -0,0 +1,657 @@
+
+        decomposed = [
+            p
+            for p in sorted(claude_skills_dir.iterdir())
+            if p.is_dir() and (p / "manifest.json").is_file() and p.name in self._CODEX_SUPPORTED_SKILLS
+        ]
+
+        for skill_dir in decomposed:
+            try:
+                codex_content = build_codex_skill_md(skill_dir)
+            except Exception:
+                continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)